src/System.Windows.Forms.Design/src/System/ComponentModel/Design/ComponentDesigner.cs

```diff
@@ -57,7 +57,8 @@ protected bool Inherited
         {
             get
             {
-                return !InheritanceAttribute.Equals(InheritanceAttribute.NotInherited);
+                InheritanceAttribute inheritanceAttribute = InheritanceAttribute;
+                return inheritanceAttribute != null && !inheritanceAttribute.Equals(InheritanceAttribute.NotInherited);
```

Comment:
nit: can you put these in a parenthetical like:

```suggestion
                return (inheritanceAttribute != null && !inheritanceAttribute.Equals(InheritanceAttribute.NotInherited));
```

This is just what I like to do when I'm returning a bool that is calculated from an expression